stdlib/public/runtime/CMakeLists.txt

```diff
@@ -192,6 +193,7 @@ add_swift_target_library(swiftImageRegistrationObjectELF
                   INSTALL_IN_COMPONENT none)
 # FIXME(compnerd) this should be compiled twice, once for static and once for
 # shared.  The static version should be used for building the standard library.
+
```

Comment:
Nit: I'm not sure these extra newlines should be here.

Reply:
Fixed.